homework/basic_html_layout/feedback.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Feedback</title>
+</head>
+<style>
+    .parag1 {
+        padding: 0;
+        margin-left: 40px;
+    }
+</style>
+
+<body>
+    <h1>My feedback form</h1>
+    <ul>
+        <li>
+            <label for="name">Name</label>
+            <input type="text" name="name"><br>
+        </li>
+
+        <li>
+            <label for="email">Email:</label>
+            <input type="text" name="email"><br>
+        </li>
+
+        <li>
+            <label for="password">Password:</label>
+            <input type="password" name="password"><br>
+        </li>
+    </ul>
+    <ul>
```

Comment:
move this `ul` inside prev `ul` and then you don't need to have `parag1 ` class.